app/jobs/order_cycle_webhook_job.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# Creates a webhook payload for an order cycle
+class OrderCycleWebhookJob < ApplicationJob
+  def perform(order_cycle_id, event)
+    order_cycle = OrderCycle.find(order_cycle_id)
```

Comment:
If the OrderCycle has been deleted, this fails and will be retried. We have that with some other job already. So maybe rescue, log and return without error.

Reply:
Good idea, done.